src/sentry/new_migrations/monkey/executor.py

```diff
@@ -2,11 +2,61 @@
 import os
 
 from django.db.migrations.executor import MigrationExecutor
+from django.db.migrations.operations import RunPython, RunSQL, SeparateDatabaseAndState
+from django.db.migrations.operations.fields import FieldOperation
+from django.db.migrations.operations.models import ModelOperation
 
 logger = logging.getLogger(__name__)
 
 
+class MissingDatabaseRoutingInfo(Exception):
```

Comment:
I'm not sure if the migration executor is the right place for this check, if there's a better place I'm happy to move it.

Reply:
I think this is a reasonable place to put it and the check for routing information.